packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -1426,6 +1456,7 @@ export const useArbTokenBridge = (
     setPendingWithdrawalMap(pendingWithdrawals)
   }
 
+  // TODO: Where used in v3?
   async function getOutboxAddress(event: L2ToL1EventResult) {
```

Comment:
I think SDK v3 takes care of the logic around which outbox to call, am I right, @yahgwai?

If that's the case, we can remove this method.